marine/marine.py

```diff
@@ -105,6 +114,11 @@ def validate_display_filter(self, display_filter: str) -> bool:
         display_filter = display_filter.encode("utf-8")
         return bool(self._marine.validate_display_filter(display_filter))
 
+    def set_preferences(self, preferences: List[str]) -> int:
```

Comment:
How complicated would it be to add a getter function to this? If it takes more than a few minutes it's not worth it, but otherwise I think it's a nice feature.

Reply:
it's can be easily done in the python code, but its gonna be only the pref's you set from the python

if i want to add this in the C i think it's gonna take a while, cause i need to save all the preferences you set in array or something, and i think its not worth it, i'm guessing also its gonna be more useful in the python code (because noeone really use the C API without the python wrap)

Reply:
Wireshark probably has a getter function for prefs (since it can display them in the gui :)) without storing them yourself. If it's easy to use then consider adding it.